common/proto-options/manager/auth_card.options

```diff
@@ -1,4 +1,5 @@
 # Options for file common/cypherock-common/proto/manager/auth_card.proto
+manager.AuthCardStatus long_names:false
```

Comment:
I think this configuration is already applied.. no need to apply again

Reply:
removed